com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -495,14 +495,18 @@ protected void TryCommitTransformToServer(Transform transformToCommit, double di
             }
             else // Non-Authority
             {
+                var position = InLocalSpace ? transformToCommit.localPosition : transformToCommit.position;
```

Comment:
Noticed this while working on parenting with multi-generation nested children (there was a bug elsewhere it turned out).  I went ahead and fixed it as I have more parenting integration tests I am working on that will be using this method when the parented NetworkObjects have a NetworkTransform too.